.github/workflows/scale-test-100-gce.yaml

```diff
@@ -0,0 +1,222 @@
+  # renovate: datasource=golang-version depName=go
+  go_version: 1.21.3
+  # Adding k8s.local to the end makes kops happy-
+  # has stricter DNS naming requirements.
+  test_name: scale-100
+  cluster_base_name: ${{ github.run_id }}-${{ github.run_attempt }}.k8s.local
+  CILIUM_CLI_MODE: helm
+
+jobs:
+  install-and-scaletest:
+    runs-on: ubuntu-latest
+    name: Install and Scale Test
+    timeout-minutes: 120
+    steps:
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Get Cilium's default values
+        id: default_vars
+        uses: ./.github/actions/helm-default
+        with:
+          image-tag: ${{ github.sha }}
+
+      - name: Set up job variables
+        id: vars
+        run: |
+          SHA="${{ github.sha }}"
+
+          # Setup Cilium install options
+          CILIUM_INSTALL_DEFAULTS="${{ steps.default_vars.outputs.cilium_install_defaults }} \
+            --wait=false"
+
+          CLUSTER_NAME="${{ env.test_name }}-${{ env.cluster_base_name }}"
+
+          echo SHA=${SHA} >> $GITHUB_OUTPUT
+          echo cilium_install_defaults=${CILIUM_INSTALL_DEFAULTS} >> $GITHUB_OUTPUT
+          echo CLUSTER_NAME=${CLUSTER_NAME} >> $GITHUB_OUTPUT
+
+      - name: Wait for images to be available
+        timeout-minutes: 30
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.SHA }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Install Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          go-version: ${{ env.go_version }}
+
+      - name: Install Cilium CLI
+        uses: cilium/cilium-cli@951eb9108277f4ffeb86d047bd492b6b96a52542 # v0.15.11
+        with:
+          release-version: ${{ env.cilium_cli_version }}
+
+      - name: Install Kops
+        uses: ./.github/actions/scale-tests/install-kops
+        with:
+          release_version: ${{ env.kops_version }}
+
+      - name: Setup gcloud credentials
+        uses: google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033 # v1.1.1
+        with:
+          credentials_json: '${{ secrets.GCP_PERF_SA_KEY }}'
+
+      - name: Setup gcloud CLI
+        uses: google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b # v1.1.1
+        with:
+          project_id: ${{ secrets.GCP_PERF_PROJECT_ID }}
+          version: "405.0.0"
+
+      - name: Clone ClusterLoader2
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: learnitall/perf-tests
```

Comment:
why your fork? We can have forks in cilium organization until the upstream repository has the patch.

Reply:
I used my fork just for convenience, and I didn't think moving to the Cilium org was a blocker. We can move it there if you'd like, but I'd prefer if we merge the workflow first so we can get some runs going and then work on migrating to the Cilium org. How does that sound?

Reply:
It is easier for others to contribute if it's part of the cilium org. I've created https://github.com/cilium/perf-tests

Reply:
I updated the workflows to point to the fork